api/_globals/crypto.json

```
@@ -0,0 +1,108 @@
+{
+  "api": {
+    "crypto": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/crypto_property",
+        "spec_url": "https://w3c.github.io/webcrypto/#dom-windoworworkerglobalscope-crypto",
+        "support": {
+          "chrome": {
+            "version_added": "37"
+          },
+          "chrome_android": {
+            "version_added": "37"
+          },
+          "deno": {
+            "version_added": "1.0"
+          },
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "11",
+            "prefix": "ms"
+          },
+          "opera": {
+            "version_added": "24"
+          },
+          "opera_android": {
+            "version_added": "24"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "5"
+          },
+          "samsunginternet_android": {
+            "version_added": "3.0"
+          },
+          "webview_android": {
+            "version_added": "37"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "deno": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
```

```suggestion
              "version_added": "10.1"
```

Confirmed in 10.1, but I can't test in 10, so verified with source:
https://trac.webkit.org/changeset/204481/webkit
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=204481